lib/util/platform_utils.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:io' show Platform;
+import 'package:flutter/foundation.dart' show kIsWeb;
+
+class PlatformUtils {
+  static bool get isMobile {
+    if (kIsWeb) {
+      return false;
+    } else {
+      return Platform.isIOS || Platform.isAndroid;
+    }
```

Comment:
We also have window platform, Can we update something like this 🙏 

```suggestion
    if (Platform.isIOS || Platform.isAndroid) {
      return true;
    } else {
      return false;
    }
```

Reply:
@Wadeewee If it is `mac` or `window` platform, the check `Platform.isIOS || Platform.isAndroid` will return `false` 😄

Reply:
@doannimble, It meets our expectations because we are using the same UI For building `web`, `macOs`, and `window desktop` platforms 🙏

Reply:
@Wadeewee This function only check it is mobile or not, so I think we can keep the current approach. It meets all our expectations too 🙏

Reply:
> @Wadeewee This function only check it is mobile or not, so I think we can keep the current approach. It meets all our expectations too 

@doannimble May I know why we need to update the UI and behavior on the `Web` and `Windows` platform 🤔 

- Here is the original version from the main branch when running on the `web` platform:

https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/b0a3c51a-d7b8-4922-b52b-f3bf8ad83963

- And here is the version from the current branch of this PR:

https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/54971177-3a8e-40d8-a970-d991238fbe16

Furthermore, The UI on the main screen was broken when resizing the screen, and here is the error:

```
======== Exception caught by rendering library =====================================================
The following assertion was thrown during layout:
A RenderFlex overflowed by 17 pixels on the bottom.

The relevant error-causing widget was:
  Column Column:file:///Users/pooh/StudioProjects/showcase-flutter-build-web-desktop/lib/screen/main/main_screen.dart:123:16
The overflowing RenderFlex has an orientation of Axis.vertical.
The edge of the RenderFlex that is overflowing has been marked in the rendering with a yellow and black striped pattern. This is usually caused by the contents being too big for the RenderFlex.

Consider applying a flex factor (e.g. using an Expanded widget) to force the children of the RenderFlex to fit within the available space instead of being sized to their natural size.
This is considered an error condition because it indicates that there is content that cannot be seen. If the content is legitimately bigger than the available space, consider clipping it with a ClipRect widget before putting it in the flex, or using a scrollable container rather than a Flex, like a ListView.

The specific RenderFlex in question is: RenderFlex#29307 relayoutBoundary=up8 OVERFLOWING
...  needs compositing
...  parentData: <none> (can use size)
...  constraints: BoxConstraints(0.0<=w<=732.0, h=105.0)
...  size: Size(732.0, 105.0)
...  direction: vertical
...  mainAxisAlignment: start
...  mainAxisSize: max
...  crossAxisAlignment: start
...  textDirection: ltr
...  verticalDirection: down
◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤
====================================================================================================
```

It seems the current version changed the design and behavior of  the `web` and `Windows` platforms and It's not a requirement of them 🙈

Reply:
@Wadeewee It is the issue of code merging. The behavior on mobile is different from web/desktop, I will create another PR to fix it.
In this PR, we will fix the main display first 🙏

Reply:
@Wadeewee FYI: Fixed at https://github.com/nimblehq/showcase-flutter-build-web-desktop/pull/59